recipe/meta.yaml

```diff
@@ -1,32 +1,36 @@
 {% set name = "pyjnius" %}
 {% set version = "1.1.3" %}
+{% set build_number = 1001 %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/kivy/pyjnius/archive/{{ version }}.tar.gz
-  sha256: 67b4031752cb32213cefd672b2792f3e75791e68bfd1ca7a5d328ff00acc1622
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #sha256: 0bc70b48eb764b20b017e44ef2df6eae94e4a2f90ae63160d2a166c406e1ccf9
+  # Currently testing on pyjnius master
+  url: https://github.com/kivy/pyjnius/archive/9e5ec4fbec01c6d7fc194cdef06a36378810b29f.zip
+  sha256: a5d6efb77ce579f3a561ad9dab214d85e3894288f38a35b1dbd274f59606359e
```

Comment:
Is the goal to still wait for a proper release to PyPi? If so, this needs to be updated.

Reply:
As far as I understand we cannot use the PyPI release because it does not ship the `pyjnius.jar` file that we need to implement Java interfaces in Python. We are still waiting for a proper tag on GithHub so we can have a proper version string and download from a GH release.

Reply:
That is exact, we will download from GH and still use the sha256. We need to wait for a git tag on the pyjnius side.